packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -0,0 +1,564 @@
+# Cart and wishlist
+
+## Introduction
+
+This document will guide you all the way to fully implement cart and wishlist modules, which will let you use these common features in your application in the best possible way. 
```

Comment:
I think this is redundant, although let @NataliaTepluhina decide. Each section could start with a similar sentence and it doesn't provide any meaningful information IMHO. The sentence below explains what this document describes.

Reply:
changed